### PR TITLE
Add README and CODEOWNERS

### PR DESCRIPTION
**Pull request by nplasterer**
This adds a readme and a codeowners file

### CODE REVIEW

**Review by rygine — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+### Run the development server
+
+```bash
+npm run dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the app.
+
```
Comment (nplasterer) [inline README.md]: @nakajima or @rygine can one of you fill this out? Not sure if this is the steps for running this locally.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Functionality
+
+- Performant Database Architecture
+- Group Chat
+- Message Reactions
+- Message Replies
+- Attachments
+  - Images
```
Comment (nplasterer) [inline README.md]: @nakajima am I missing any functionality?

File: README.md (hunk 1 above)
Comment (rygine) [reply README.md]: yep, that's the correct command for local dev

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+npm install
+```
+
+## Run the development server
+
+```bash
+npm run dev
+```
+
+Open [http://localhost:5173](http://localhost:5173) in your browser to access the XMTP React playground app.
```
Comment (jhaaaa) [inline README.md:34]: @nplasterer Just an FYI that I updated the port number from 3000 to 5173 based on what opened when I ran this command. Do we want to update the code to make it open on port 3000? <3

File: README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# XMTP React playground
+
```
Comment (jhaaaa) [inline README.md:2]: @nplasterer Do you think playgrounds need alpha/beta/production designations?

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (jhaaaa) [inline LICENSE:1]: @nplasterer removed the reference to the licensing and maintenance in the README and added this standard LICENSE file. This is the one we use in xmtp-ios.

**Review by jhaaaa — APPROVED** [review]
So excited about this playground tool, @nplasterer! TY!

Made from tech writerly edits to the README - mainly to reinforce the role of the playground app as a tool for devs to build with.

File: README.md (hunk 4 above)
Comment (nplasterer) [reply README.md:2]: I think playgrounds should be dev/local. We probably shouldnt be encouraging production usage in a playground.

File: README.md (hunk 3 above)
Comment (nplasterer) [reply README.md:34]: This is the port it opened for me too so this seems fine!

**Pull request merged**